<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaClasse</class>
 <widget class="QMainWindow" name="MaClasse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaClasse</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horiLayout" stretch="1,0">
      <item>
       <widget class="QOpenGLWidget" name="openGLWidget">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="16" column="0" colspan="2">
         <widget class="QPushButton" name="startButton">
          <property name="styleSheet">
           <string notr="true">color: green; background: grey; font: bold 20px</string>
          </property>
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="defrostButton">
          <property name="text">
           <string>DEFROST</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0" colspan="2">
         <widget class="QPushButton" name="stopButton">
          <property name="styleSheet">
           <string notr="true">color: red; background: grey; font: bold 20px;
</string>
          </property>
          <property name="text">
           <string>STOP</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="clockButton">
          <property name="text">
           <string>CLOCK</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QDial" name="dial"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;b&gt;CURRENT STATUS:&lt;b&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="powerButton">
          <property name="text">
           <string>POWER</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLCDNumber" name="time">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="modeButton">
          <property name="text">
           <string>MODE</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QFrame" name="gridFrame">
          <property name="styleSheet">
           <string notr="true">background: white; border-style:outset; border-width: 2px</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="5" column="0">
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">border-style: none;color: black
</string>
             </property>
             <property name="text">
              <string>&lt;i&gt;Mode 3:&lt;i&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">color: black; background: white; border-style: none</string>
             </property>
             <property name="text">
              <string>&lt;i&gt;Mode 1:&lt;i&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">border-style: none;color: black
</string>
             </property>
             <property name="text">
              <string>&lt;i&gt;Mode 2:&lt;i&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">border-style: none; color: black
</string>
             </property>
             <property name="text">
              <string>Microwave</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">border-style: none;color: black
</string>
             </property>
             <property name="text">
              <string>Grill</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_9">
             <property name="styleSheet">
              <string notr="true">border-style: none;color: black
</string>
             </property>
             <property name="text">
              <string>&lt;i&gt;Mode 4:&lt;i&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true">border-style: none;color: black
</string>
             </property>
             <property name="text">
              <string>Pizza</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_10">
             <property name="styleSheet">
              <string notr="true">border-style: none;color: black
</string>
             </property>
             <property name="text">
              <string>Oven</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">color: black; background: white; border-style: none</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;MODES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
